contracts/staking/contracts/src/libs/LibStakingRichErrors.sol

```diff
@@ -488,4 +505,21 @@ library LibStakingRichErrors {
             endEpoch
         );
     }
+
+    function StorageLayoutError(
```

Comment:
I'm assuming this doesn't get included in the bytecode and is purely for reference, right? I'm on board.

Reply:
This was actually just an accident 🙈